languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -32034,10 +32032,10 @@ though, and has since been slightly modified)-->
                 </marker>
                 <token regexp="yes">correctifs?</token>
             </pattern>
-            <message>« Identifier des correctifs » est considéré comme un anglicisme. En voici des alternatives :</message>
+            <message>« Identifier des correctifs » peut être considéré comme un anglicisme.</message>
             <suggestion><match no="1" postag="(V.*)" postag_regexp="yes" postag_replace="$1">proposer</match></suggestion>
             <suggestion><match no="1" postag="(V.*)" postag_regexp="yes" postag_replace="$1">suggérer</match></suggestion>
-            <example correction="proposer|suggérer"><marker>identifier</marker> des correctifs</example>
+            <example correction="proposer|suggérer">Il a <marker>identifier</marker> des correctifs lors de la conférence de presse.</example>
```

Comment:
Maybe this example should not contain a second error (Il va/veut?)

Reply:
That's corrected.